.gitignore

```diff
@@ -1 +1,3 @@
 target/
+.idea
```

Comment:
Are these additions intended? IMO, these ignores should be local. See how to implement them globally in your computer in https://help.github.com/articles/ignoring-files/.